src/Libraries/CoreNodeModelsWpf/CoreNodeModelsWpf.csproj

```diff
@@ -25,9 +25,9 @@
     </ReferenceCopyLocalPaths>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <PackageReference Include="LiveCharts" Version="0.9.7" />
-    <PackageReference Include="LiveCharts.Wpf" Version="0.9.7" />
     <PackageReference Include="DotNetProjects.Extended.Wpf.Toolkit" Version="5.0.103" />
+    <PackageReference Include="LiveChartsCore" Version="2.0.0-beta.855" />
```

Comment:
Any binary diff?

Reply:
Since the VS2022 build is failing, the bindiff job is failing as well, the PR does bring in some Livecharts2 and skiasharp binaries.